lib/node/storage.js

```diff
@@ -1,15 +1,98 @@
 /* eslint no-unused-vars: 0 */
+import { readFile, writeFile } from "fs";
+import * as lockfile from "proper-lockfile";
+import * as path from "path";
 
-export const canStoreURLs = false;
+export const canStoreURLs = true;
+const defaultStoragePath = path.join(process.cwd(), "tus-url-storage.json");
 
-export function setItem(key, value) {
+export class FileStorage {
+  constructor(filePath) {
+    this.path = filePath || defaultStoragePath;
```

Comment:
I don't think there should be a default storage path. Instead a user should always explicitly name the path that should be used.